src/ServicePulse.Host/vue/src/views/EventsView.vue

```diff
@@ -13,28 +13,29 @@ const pageModel = ref<DataViewPageModel<EventLogItemType>>({ data: [], totalCoun
 </script>
 
 <template>
-  <LicenseExpired />
-  <template v-if="!licenseStatus.isExpired">
-    <ServiceControlNotAvailable />
-    <template v-if="connectionState.connected">
-      <div class="events events-view">
-        <DataView api-url="eventlogitems" v-model="pageModel" :auto-refresh-seconds="5" :show-items-per-page="true" :items-per-page="20">
-          <template #data>
-            <div class="row">
-              <div class="col-sm-12">
-                <h1>Events</h1>
-                <EventLogItem v-for="item of pageModel.data" :eventLogItem="item" :key="item.id" />
-              </div>
+  <div class="container">
```

Comment:
On other pages, the container is inside the license expired template

Reply:
Fixed @mikeminutillo